### DISCUSSION TITLE
Check if I'm on 404 error page

### DISCUSSION THREAD

**Discussion by pathly-dev** [statamic/cms#4331] [Q&A] [answered]
How do I check if I'm on the 404 error page?
I have elements I want to have on every page except on the error page. So I put it in the layout.html, but I dont want to show that on the error page. I tried something like {{ if 404 }} like somewhere was recommended, but nothing is working.

**Comment by duncanmcclean** [comment]
You can probably check your page's template, like this:

```
{{ if current_template !== 'errors/404' }}
<!-- show your stuff -->
{{ /if }
```

**Reply by pathly-dev** [reply]
> You can probably check your page's template, like this:
> 
> ```
> {{ if current_template !== 'errors/404' }}
> 
> {{ /if }
> ```

I was looking for something like this, but unfortunately it seems like it doesn't work with the 404 template

**Reply by pathly-dev** [reply]
I thought maybe the template name is wrong. But when I try to print out {{ template }} (like you can find in the statamic docs) it doens't print out anything

**Comment by jasonvarga** [comment, marked as answer]
Like Duncan said, but use a dot. `{{ if current_template !== "errors.404" }}`

You can always check it by doing `{{ current_template }}`.

**Reply by pathly-dev** [reply]
Yes that's working. Thanks!
